File: Actividad P1A9 ToString/Auto.py
class Auto:

    def __init__(self):
        self.__modelo = 0
        self.marca = ""

    @property
    def modelo(self):
        return self.__modelo

    @modelo.setter
    def modelo(self, value):
        self.__modelo = value

#def imprimir(self):
    #	print("\nModelo:", self.__modelo, "\nMarca:", self.marca)

    def __str__(self):
        return f"\nModelo: {self.__modelo} \nMarca: {self.marca}"

#return "\nModelo:", str(self.__modelo), "\nMarca:", self.marca 
#str() es para convertir otros valores a string, tambien puedes usar f de formato para lo mismo 

File: Actividad P1A9 ToString/Auto.cs
using System;

public class Auto {

	private int modelo;
	public string marca;

	public int Modelo {
		get { return this.modelo; }
		set { this.modelo = value; }
	}

	/*public void imprimir() {
		Console.WriteLine("\nModelo: " + this.modelo + "\nMarca: " + this.marca);
	}*/
	
	public override string ToString(){
	    return $"\nModelo: " + this.modelo + "\nMarca: " + this.marca; //$ es como la f de formato 
	}
	
}